database_connector.py

```diff
@@ -0,0 +1,33 @@
+import pymysql.cursors
+
+
+class DatabaseConnector:
+    def __init__(self, database=None):
+        self.host = 'localhost'
+        self.user = 'root'
+        self.password = ''
+        self.db = database
+        self.charset = 'utf8mb4'
+        self.cursorclass = pymysql.cursors.DictCursor
+
+    def set_database(self, database):
+        self.db = database
+
+    def get_connection(self):
```

Comment:
If this is private, prefix with _

Reply:
neither of these are private currently, but they might become and will update if so